modules/apps/transfer/keeper/relay.go

```diff
@@ -271,9 +251,7 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 				return errorsmod.Wrapf(err, "failed to send coins to receiver %s", receiver.String())
 			}
 
-			denomPath := token.Denom.Path()
-			labels = append(labels, telemetry.NewLabel(coretypes.LabelSource, "false"))
-			defer internaltelemetry.ReportOnRecvPacketTelemetry(transferAmount, denomPath, labels)
+			defer telemetry.ReportOnRecvPacket(packet.GetSourcePort(), packet.GetSourceChannel(), token)
```

Comment:
can deduplicate even further. 

```go
If {

} else {

}
defer telemetry. ReportOnRecvPacket(...)
```

currently we execute the same defer in the if and else